data/src/main/java/com/microsoft/azure/kusto/data/auth/ConnectionStringBuilder.java

```diff
@@ -77,6 +153,34 @@ public ConnectionStringBuilder(ConnectionStringBuilder other) {
         this.applicationNameForTracing = other.applicationNameForTracing;
     }
 
+    // NewConnectionStringBuilder Creates new Kusto ConnectionStringBuilder.
```

Comment:
Please use a proper Javadoc comment - this can't be seen by the user. (see other places where we use /** )